### ISSUE TITLE
Is gogs dead?

### ISSUE THREAD

**Issue by jsieler** [gogits/gogs#1304]
Is the project mainly dead? Since over a month there is nothing new... Anyone knows?

**Comment by fuxiaohei** [comment]
we are waiting for @Unknwon coming back from US. He's a student.

If something gets worse, we will make new plan for gogs in June or July.

**Comment by tboerger** [comment]
@fuxiaohei accepting for contributors for accepting pull requests would already help. Or just opening the contribution to the long missing feature for pull requests :)

**Comment by fredghostkyle** [comment]
@fuxiaohei Do we have a ETA to when he's back? Also, just because you're missing your leader doesn't mean nothing gets done. That might sound mean and really harsh but it's true.

**Comment by tboerger** [comment]
If more people would have write access, yes... We will get an answer really pretty soon, i think tomorrow.

**Comment by vheuken** [comment]
@tboerger I don't see why that prevents work from getting done. Why can't someone just fork it and merge pull requests to that fork, then merge that when he gets back?

**Comment by tboerger** [comment]
https://github.com/go-gitea/gitea

**Comment by srvg** [comment]
What's the deal with go-gitea exactly? A real fork? The project repo doesn't even mention it's a fork from here. What are your intentions @tboerger ?

Unrelated: can we have a mailing list and an official irc room? Github issues s**k for support stuff.

**Comment by tboerger** [comment]
go-gitea will mention that, just give us sone time. There is already the official Gitter chat :)

**Comment by rojaro** [comment]
@srvg ... check https://gitter.im/gogits/gogs

**Comment by redelman** [comment]
Since gogs has been forked to go-gitea, does that mean that gogs is indeed dead? Wondering if I should bother updating my gogs at work to go-gitea.

**Comment by tboerger** [comment]
@redelman we need a little bit more time to get a new stable release of gitea out of the door, afterwards it makes sense.

**Comment by iraycd** [comment]
This topic is trending on Hacker News. https://news.ycombinator.com/item?id=9708832

**Comment by jh86** [comment]
It seems pretty ridiculous that people are complaining about a lack of updates for free (as in beer) software. "Be the change you wish to see in the world", fork the project, make the changes you need, and stop waiting for someone else to do it. Unless I'm misunderstanding something here -- are you guys paying his salary?

**Comment by tboerger** [comment]
Nobody earns money with gogs/gitea. And it looks like some people on HN thought that i have forked gogs, that's not true, I just started to dive into contribution ;)

**Comment by StrangeWill** [comment]
@jh86 It isn't about @Unknwon coming in and working for free, it's about one of the project maintainers continuing the project -- sure Gogs can (and has been) forked. There is just little reason for a project to fall off the face of the earth because maintainers are all waiting on a student to come back before touching it.

I'm glad that the fork is happening -- I was considering having to ditch Gogs over the lack of active development (and am still kind of patchy on that), it's solid software and I'd like to continue contributing to it.

**Comment by f0086** [comment]
A month later an still no sign of life here.
Is go-gitea the official fork where the active development happen now?
Time to fold up here?

**Comment by tboerger** [comment]
@f0086 as we already said, yes it is the official fork. If @unknwon ever comes back we want to merge the changes back to gogs of course.

**Comment by bclermont** [comment]
@tboerger he's back https://github.com/gogits/gogs/commit/4fca548ed11cefedcaa595be1c08625a9ae9f878

**Comment by tboerger** [comment]
@bclermont interesting, now let's see if he is open for contributions and maybe write access :)

**Comment by bseng** [comment]
I really hope they merge as gitea documentation always link back to gogs

**Comment by bclermont** [comment]
I wonder why gitea fork on first commit was to change project name. this will make all merging process harder.

**Comment by tboerger** [comment]
A quote from @lunny on the gitea Gitter chat:

> I have contacted with Unknwon. And he said he will consider to merge gitea back to gogs if the progress is not difficult. And maybe he want to keep gogs simple and he don't want to add more people as contributors.
> So I think we can continue gitea and no need to wait for him.

**Comment by f0086** [comment]
What is going on with you @lunny , @fuxiaohei and @slene ? Do you have write access to the repo? There are open 18 pull requests at the moment (and 333 open Issues!). If nobody review/merge the pull requests and the issues gets ignored, the project is doomed in the long term.

**Comment by tboerger** [comment]
@f0086 as already mentioned above, @Unknwon is the only one that have write access to gogs, and this will stay like it is. @lunny and @fuxiaohei created the fork https://github.com/go-gitea/gitea where currently 4 people got write access. It won't take too long, than we will have finished pull requests, wikis are already done, than we will release gitea v0.7

**Comment by linquize** [comment]
@Unknwon is back recently, but unsure if he knows the fork go-gitea/gitea

**Comment by tboerger** [comment]
@linquize just read https://github.com/gogits/gogs/issues/1304#issuecomment-119106551

**Comment by stephen-hill** [comment]
@tboerger 

Is there an estimated release date for v0.7?

**Comment by tboerger** [comment]
@stephen-hill no estimation, but we need to finish the pull request feature first.

**Comment by thoroc** [comment]
+1

**Comment by flomine** [comment]
Hi all contributors of gogs and gitea.
Do you consider to merge gitea back to gogs or not?

The situation is not clear to me and I think this situation is not good for gogs and gitea ?
Do we have to choose one now ?
Please let me know.

**Comment by HLFH** [comment]
Hi. IMHO, gitea has wiki support, not gogs. I'm waiting for this feature. So, as soon as gitea is released https://github.com/go-gitea/gitea/issues/22 I'll migrate to gitea, which seems to have a more open governance.

**Comment by f0086** [comment]
@flomine Exactly! If this gogs vs. gitea struggle is going on for another month, I will switch to a completely other platform. This is just unprofessional and ruin the project.

**Comment by tboerger** [comment]
@f0086 if you think this is so unprofessional have you ever managed such a complicated situation? Just criticize is pretty easy. Suggest improvements is much more complicated...

We simply have no write access and will not get any write access to gogs. So multiple peaople of us want to push gitea of course. Merging features back will be pretty difficult as nobody knows what gets rejected or accepted by @Unknwon as he only reacts rarely to any request...

Currently we are in the middle of the summer, so many people are doing vacation or relax within the summer. Gitea nor gogs are full time projects, nobody gets payed for that, so this is as usual within opensource projects that things take time.

Just relax, gitea will be released as soon as pull requests are finished. You want to contribute something? Go ahead, we are happy to see more contributions! there are enough open issues with lowhanging fruits that can get fixed within the upcoming 0.7 release. Every bigger change can get into 0.8 ;)

**Comment by Unknwon** [comment]
Apologize for being away for so long. :flushed:

To finish this thread, there are some words I would like to say:

- Gogs isn't dead, and it's my bad for being inactive for 3 months. There are reasons cannot be told and understood. 
- The initial reason that I removed write access of @lunny and @fuxiaohei was there is a build failure after merged a PR, and I got e-mail notified. And the reason access was not giving back was that I wasn't able to have enough time review all PRs, and I don't like unfamiliar or not totally understood code is merged into codebase. I believe it's better to keep it as it was and merge when things get to normal like now.
- In my point of view, it's a sign of success of Gogs that Gitea forked it. Gogs is under MIT license and there is no problem with me totally that Gitea is developing its own version. It happens often in open source community(when you are not satisfied with upstream version, I fork a lot actually).
- Gitea won't be merged back to Gogs, it's not about merging work is huge and hard, it's the differences of fundamental philosophy. I personally do not like to push hard to release new features, but make code neat and clean, it's not good for business, but Gogs isn't a business, making it is what I love to do.

Thank you all, and happy coding!
